src/frontend/templates/index.html

```diff
@@ -190,6 +193,13 @@ <h4 class="card-table-header">No Transactions Found</h4>
                             <td class="transaction-account">
                               {{ t.fromAccountNum }}
                             </td>
+                            <td class="transaction-label">
+                              {% if t.accountLabel.strip() != "" %}
```

Comment:
nit: it may be cleaner to None and `{% if t.accountLabel.strip() != None %}` instead of an empty string here

Reply:
Alrighty, changed to using `None`